config/table_definitions.yaml

```diff
@@ -0,0 +1,155 @@
+define:
+  global_min_year: &global_min_year 1999
+
+# Table definitions containing the possible values to use for each table job,
+# as well as table-level schema overrides per column. Null values indicate
+# the column is not applicable to the table, or that there should be no
+# check on the possible value
+tables:
+  aasysjur:
+    min_year: null
+    has_cur: false
+    has_parid: false
+  addn:
+    min_year: *global_min_year
```

Comment:
I plan to use a non-null `min_year` value here implicitly to check if tables have a `taxyr` column, but will save that for another PR.